const promiseMsg = () => new Promise((resolve, reject) => {
  const error = false;

  if (!error)
    return setTimeout(() => resolve('Message: ğŸ¤¡'), 2000); // Message: ğŸ¤¡ <-- after 2 seconds

  return reject(new Error('Message: ğŸ‘¿'));
});
const clown = async () => {
  console.log('Before: ğŸ¤¡');

  try {
    console.log(await promiseMsg());
  } catch (err) {
    console.log(err);
  }

  console.log('After: ğŸ¤¡');
};

clown();
